queries/currency.py

```diff
@@ -205,10 +226,17 @@ def help_text(lemma):
 
 QCurCAD/fall →
     | 'kanadískur:lo'_kk/fall 'dollari:kk'/fall
+    | 'kanadadalur:no'_kk/fall
+    | 'kanadadollari:no'_kk/fall
+    | "kanadadollar" # Common mistake
     | currency_cad/fall
 
+QCurCAD_nf →
+    "kanadadalur" | "kanadadalurinn"
+    | "kanadadollari" | "kanadadollarinn"
+
 QCurCAD_þgf →
-    "kanadadal" | "kanadadalnum"
+    "kanadadal" | "kanadadalnum" | "kanadadollar"
```

Comment:
No need to repeat `"kanadadollar"` since it's already been defined in `QCurCAD/fall`

Reply:
Haha, I was planning to make a few commits before pressing  "Request review" but you've jumped the gun! :P